### PR TITLE
DDCE-1060: Change straight apostrophes to curly apostrophes

### PR DESCRIPTION
**Pull request by abhijeetardale**
# DDCE-1060

##Bug fix / New feature (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

**Comment by abhijeetardale** [comment]
Linked PR please
https://github.com/hmrc/charities-registration-journey-tests/pull/120

File: conf/messages.en [hunk 1]
```diff
@@ -1017,23 +1017,23 @@ addAnotherOtherOfficial.error.required = Select yes if you want to add another o
...
 # Charity Nominee messages
 # ----------------------------------------------------------
-charityNominee.title = The charity''s nominee
-charityNominee.heading = The charity''s nominee
+charityNominee.title = The charity’s nominee
+charityNominee.heading = The charity’s nominee
 charityNominee.p1 = You now have the option of adding a nominee.
 charityNominee.p2 = A nominee is a person or an organisation from outside the charity who can submit Gift Aid or other tax repayment claims on your behalf. They might also receive the repayments too.
-charityNominee.p3 = Nominees aren''t ''managers'' for the purposes of the fit and proper persons test. But you do have to tell us if the charity appoints a nominee or the nominee changes.
+charityNominee.p3 = Nominees aren’t ’managers’ for the purposes of the fit and proper persons test. But you do have to tell us if the charity appoints a nominee or the nominee changes.
```
Comment (robert-webb-hmrc) [inline conf/messages.en:1036]: This apostrophe is not oriented correctly, needs to be flipped
Screenshot 2020-09-02 at 16 46 26

File: conf/messages.cy [hunk 2]
```diff
@@ -1017,23 +1017,23 @@ addAnotherOtherOfficial.error.required = (W)Select yes if you want to add anothe
...
 # Charity Nominee messages
 # ----------------------------------------------------------
-charityNominee.title = (W)The charity''s nominee(W)
-charityNominee.heading = (W)The charity''s nominee(W)
+charityNominee.title = (W)The charity’s nominee(W)
+charityNominee.heading = (W)The charity’s nominee(W)
 charityNominee.p1 = (W)You now have the option of adding a nominee.(W)
 charityNominee.p2 = (W)A nominee is a person or an organisation from outside the charity who can submit Gift Aid or other tax repayment claims on your behalf. They might also receive the repayments too.(W)
-charityNominee.p3 = (W)Nominees aren''t ''managers'' for the purposes of the fit and proper persons test. But you do have to tell us if the charity appoints a nominee or the nominee changes.(W)
+charityNominee.p3 = (W)Nominees aren’t ’managers’ for the purposes of the fit and proper persons test. But you do have to tell us if the charity appoints a nominee or the nominee changes.(W)
```
Comment (robert-webb-hmrc) [inline conf/messages.cy:1036]: Please see EN messages

**Review by robert-webb-hmrc — CHANGES_REQUESTED** [review]
Could this also be rebased please

**Comment by abhijeetardale** [comment]
fixed the commets and done the rebase

**Pull request merged**